docs/ReleaseHistory.md

```diff
@@ -22,6 +22,7 @@
 - BRK: `WellKnownRegexPatterns.*` properties are now of stronger type`IReadOnlyList<RegexPattern>` instead of `IEnumerable<RegexPattern>`.
 - BRK: `SecretMasker.SyncObject` is now a read-only property instead of a public mutable field.
 - NEW: `SecretMasker` is now capable of finding identifiable keys without relying on delimiting characters`.
+- NEW: Add `RegexPattern.CreatedVersion` and `RegexPattern.LastUpdatedVersion` to track version of rule introduction and last update to rule logic. This change has the result of emitting this information to the generated rules JSON metadata files.
```

Comment:
nit: "This change has the result of emitting this information to ..." is a bit wordy. Perhaps "These versions are emitted to ..."?

Reply:
nice yes.